.github/workflows/build.yml

```diff
@@ -33,6 +33,12 @@ jobs:
       - name: Build with cx_Freeze
         run: python torrent-client/setup.py build
 
+      - name: Install bsdtar
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libarchive-tools
```

Comment:
You could move these commands to the Linux block below, so it all stays together